test/parallel/test-child-process-escape-argument.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+const { escapeArgument } = require('child_process');
+assert.strictEqual(escapeArgument('foo'), 'foo');
+assert.strictEqual(escapeArgument('foo bar'), "'foo bar'");
+assert.strictEqual(escapeArgument('foo & bar'), "'foo & bar'");
+assert.strictEqual(escapeArgument('foo & bar$'), "'foo & bar$'");
+assert.strictEqual(escapeArgument('foo%bar'), 'foo%bar');
```

Comment:
I suggest adding a couple of tests based on regexs: 
```suggestion
assert.strictEqual(escapeArgument('foo%bar'), 'foo%bar');
assert.strictEqual(escapeArgument('FOO'), 'FOO', 'Uppercase alphanumeric string');
assert.strictEqual(escapeArgument('foo42'), 'foo42', 'Alphanumeric string with numbers');
assert.strictEqual(escapeArgument('foo_42'), 'foo_42', 'Alphanumeric string with underscore');
assert.strictEqual(escapeArgument('123'), '123', 'Numeric string');
assert.strictEqual(escapeArgument('@#$'), '\'\@\#\$\'' , 'Special characters string');
assert.strictEqual(escapeArgument('`\''), '\'\`\'', 'String with backtick and single quote');
```